moveit_setup_assistant/src/widgets/collision_matrix_model.h

```diff
@@ -0,0 +1,80 @@
+/*********************************************************************
```

Comment:
I thought about that during development too. But I decided it more strongly belongs to the associated widget - these model classes are not a tool on its own.